src/languages/pyret/ast.js

```diff
@@ -101,15 +106,19 @@ export class Func extends ASTNode {
   }
 
   pretty() {
-    return P.lambdaLikeSexpr(this.id, this.args, this.body);
+    return P.horzArray([P.txt("fun "), this.name, P.txt("("), P.horzArray(this.args.map(p => p.pretty())), P.txt(")"), this.body]);
   }
 
   render(props) {
+    // TODO: uncommenting this expression leads to using an object instead of a react element
+    let args = /* this.args[0].reactElement() */ undefined;
```

Comment:
@justinpombrio ?

Reply:
That line is doing the right thing type-wise. Something else is going on: maybe `args` is not an array of `Bind`?

Reply:
(Preston just confirmed: issue's in bind.id)